parquet-protobuf/src/main/java/org/apache/parquet/proto/ProtoConstants.java

```diff
@@ -26,6 +26,7 @@ public final class ProtoConstants {
   public static final String METADATA_ENUM_PREFIX = "parquet.proto.enum.";
   public static final String METADATA_ENUM_KEY_VALUE_SEPARATOR = ":";
   public static final String METADATA_ENUM_ITEM_SEPARATOR = ",";
+  public static final String CONFIG_IGNORE_UNKNOWN_FIELDS = "parquet.proto.ignore.unknwon.fields";
```

Comment:
```suggestion
  public static final String CONFIG_IGNORE_UNKNOWN_FIELDS = "parquet.proto.ignore.unknown.fields";
```

Reply:
Could you please add a comment for `CONFIG_IGNORE_UNKNOWN_FIELDS`? Like what we did for `CONFIG_ACCEPT_UNKNOWN_ENUM`.